### DISCUSSION TITLE
NextJs 13.4.0 has two new processes (jest-worker/processChild) running in NODE_ENV=production

### DISCUSSION THREAD

**Discussion by S-YOU** [vercel/next.js#49238] [Help] [answered]
### Summary

NextJs 13.4.0 has two new processes running in NODE_ENV=production. Is that intentional?

![image](https://user-images.githubusercontent.com/1001175/236312679-4f46e5a1-6aaa-43c2-8a4c-18fac236c29c.png)

Related code

```js
              addToTracedFiles(
                '',
                require.resolve('next/dist/compiled/jest-worker/processChild')
              )
              addToTracedFiles(
                '',
                require.resolve('next/dist/compiled/jest-worker/threadChild')
              )
            }
```

**Comment by csi-lk** [comment]
Seeing this too coming through instrumentation logs as part of fetch calls which is strange

**Comment by switz** [comment]
can anyone provide some insight into this?

**Comment by leerob** [comment, marked as answer]
Closing and moving the conversation to https://github.com/vercel/next.js/issues/45508
